lib/gettext_i18n_rails/model_attributes_finder.rb

```diff
@@ -63,7 +63,10 @@ def model_attributes(model, ignored_tables, ignored_cols)
     end
 
     def models
-      if Rails::VERSION::MAJOR > 2
+      if Rails::VERSION::MAJOR > 4
+        Rails.application.eager_load! # make sure that all models are loaded so that direct_descendants works
+        ApplicationRecord.direct_descendants
```

Comment:
yeah that sounds safe

On Thu, Jan 14, 2016 at 11:00 PM, Eduardo Bautista <notifications@github.com
> wrote:

> In lib/gettext_i18n_rails/model_attributes_finder.rb
> <https://github.com/grosser/gettext_i18n_rails/pull/148#discussion_r49743875>
> :
>
> > @@ -63,7 +63,10 @@ def model_attributes(model, ignored_tables, ignored_cols)
> >      end
> >
> >      def models
> > -      if Rails::VERSION::MAJOR > 2
> > +      if Rails::VERSION::MAJOR > 4
> > +        Rails.application.eager_load! # make sure that all models are loaded so that direct_descendants works
> > +        ApplicationRecord.direct_descendants
>
> Hmmm, a sum of ApplicationRecord.direct_descendants +
> ActiveRecord::Base.direct_descendants - [ApplicationRecord] maybe?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/gettext_i18n_rails/pull/148/files#r49743875>.
>